pyzeebe/channel/__init__.py

```diff
@@ -1,3 +1,10 @@
-from pyzeebe.channel.camunda_cloud_channel import create_camunda_cloud_channel
+from pyzeebe.channel.camunda_cloud_channel import (
+    create_camunda_cloud_channel,  # FIXME: could be removed
```

Comment:
Left it in so the "old" way for channel creation as default. 
For the usage of the new oauth channels they would have to be imported with e.g. `from pyzeebe.channel.oauth_channel import create_oauth2_client_credentials_channel`.

If we want to have oauth as the default, we could uncomment the lines below and phase out the old way. After that we could delete the "old" way altogether.

Reply:
I'm not familiar with Camunda Cloud. Let us choose only one way to create a channel for it.

Reply:
The new way would be more consistent.